_posts/2018-12-12-cockpit-184.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cockpit 184
+author: mvollmer
+date: 2018-12-12
+tags: cockpit linux
+slug: cockpit-184
+category: release
+summary: Cockpit with machines and storage improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 184.
+
+### Machines: Dialog and tab layout is now responsive
+
+### Storage: Filesystem labels are validated up front
+
+### Storage: Some mount options are prefilled when needed
+
+### Cockpit on your Desktop (preview)
```

Comment:
How about that:

### Integration of Cockpit pages on the desktop

For some use cases it may be desirable to use a particular Cockpit page on a desktop, if a native application does not exist. This can be done with the new "cockpit-desktop" tool, which runs a Cockpit web server and a web browser (Webkit, Firefox, or Chromium/Chrome) in an isolated network namespace. This avoids having to enable Cockpit system-wide and having to log into it through the web browser again.

cockpit-desktop may be used by system integrators or local administrators through a standard [desktop launcher](https://standards.freedesktop.org/desktop-entry-spec/latest/). Cockpit ships no pre-defined launchers by itself.